Lesson006/Task 3 HW06.ArrayModifier/Program.cs

```diff
@@ -1,39 +1,45 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
-namespace Task_2_HW06.ThirdArrayCreator
+namespace Task3HW06.ArrayModifier
 {
     internal class Program
     {
         static void Main(string[] args)
         {
-            int[] arrayRandomInput = new int[5];
-            Random random = new Random();
+            Console.Write("Enter the number of array elements:\t");
 
-            for (int i = 0; i < arrayRandomInput.Length; i++)
+            int elementsCount = int.Parse(Console.ReadLine());
+            int[] myArrey = new int[elementsCount];
+
+            for (int i = 0; i < elementsCount - 1; i++)
             {
-                arrayRandomInput[i] = random.Next(20);
-                Console.WriteLine("Array element value " +  " = " + arrayRandomInput[i]);
+                myArrey[i] = int.Parse(Console.ReadLine());
             }
 
-            int[] arrayKeyboardInput = new int[5];
+            int lastNumber = 0;
+
+            Console.Write("Enter last value: ");
+            lastNumber = Convert.ToInt32(Console.ReadLine());
+
+            myArrey[myArrey.Length - 1] = lastNumber;
+
+            int rightShift = myArrey[myArrey.Length - 1];
+
+            for (int i = myArrey.Length - 1; i > 0; i--) //How to rotate an array to the left?
+            {
+                myArrey[i] = myArrey[i - 1];
+            }
```

Comment:
You have to complete part of the task. To shift the elements, we need to place the values starting from the specified index into a new array and then reassign starting from the new index. This is one of the solutions.